iceoryx_posh/test/integrationtests/test_service_discovery.cpp

```diff
@@ -55,316 +107,314 @@ TEST_F(RoudiFindService_test, OfferSingleMethodServiceSingleInstance)
 }
 
 
-TEST_F(RoudiFindService_test, OfferServiceWithDefaultServiceDescriptionFails)
+TEST_F(ServiceDiscovery_test, OfferServiceWithDefaultServiceDescriptionFails)
 {
     ::testing::Test::RecordProperty("TEST_ID", "1db1ce50-4e95-46f3-8682-9cc90576dbc0");
-    auto isServiceOffered = senderRuntime->offerService(iox::capro::ServiceDescription());
+    auto isServiceOffered = sut.offerService(iox::capro::ServiceDescription());
     this->InterOpWait();
 
     ASSERT_EQ(false, isServiceOffered);
 }
 
-TEST_F(RoudiFindService_test, OfferServiceWithValidEventIdSucessfull)
+TEST_F(ServiceDiscovery_test, OfferServiceWithValidEventIdSucessfull)
 {
     ::testing::Test::RecordProperty("TEST_ID", "1107d0e3-42e1-4b24-9a4d-cef8badb7154");
-    auto isServiceOffered = senderRuntime->offerService({"service1", "instance1", "event1"});
+    auto isServiceOffered = sut.offerService({"service1", "instance1", "event1"});
     this->InterOpWait();
 
     ASSERT_EQ(true, isServiceOffered);
 }
 
-TEST_F(RoudiFindService_test, OfferServiceWithInvalidEventIdFails)
+TEST_F(ServiceDiscovery_test, OfferServiceWithInvalidEventIdFails)
 {
-    ::testing::Test::RecordProperty("TEST_ID", "b67b4990-e2fd-4efa-ab5d-e53c4ee55972");
-    auto isServiceOffered =
-        senderRuntime->offerService({"service1", iox::capro::InvalidIdString, iox::capro::InvalidIdString});
+    auto isServiceOffered = sut.offerService({"service1", iox::capro::InvalidIdString, iox::capro::InvalidIdString});
     this->InterOpWait();
 
     ASSERT_EQ(false, isServiceOffered);
 }
 
-TEST_F(RoudiFindService_test, ReofferedServiceWithValidServiceDescriptionCanBeFound)
+TEST_F(ServiceDiscovery_test, ReofferedServiceWithValidServiceDescriptionCanBeFound)
 {
-    ::testing::Test::RecordProperty("TEST_ID", "6e3af6f8-7798-4887-8526-f797068492ba");
-    EXPECT_TRUE(senderRuntime->offerService({"service1", "instance1", "event1"}));
+    ::testing::Test::RecordProperty("TEST_ID", "b67b4990-e2fd-4efa-ab5d-e53c4ee55972");
+    EXPECT_TRUE(sut.offerService({"service1", "instance1", "event1"}));
     this->InterOpWait();
-    EXPECT_TRUE(senderRuntime->stopOfferService({"service1", "instance1", "event1"}));
+    EXPECT_TRUE(sut.stopOfferService({"service1", "instance1", "event1"}));
     this->InterOpWait();
-    EXPECT_TRUE(senderRuntime->offerService({"service1", "instance1", "event1"}));
+    EXPECT_TRUE(sut.offerService({"service1", "instance1", "event1"}));
     this->InterOpWait();
```

Comment:
The last wait before `findService` I understand, the others are redundant or considered part of the test itself (i.e. not only due to the delay of registering a service). Can be kept for now but the test intention should be reconsidered after refactoring the functionality I think.